src/utils/cpuProfileParsing.ts

```diff
@@ -0,0 +1,119 @@
+import _ from 'lodash';
+import {
+  HermesSample,
+  HermesStackFrame,
+} from '../types/hermesProfileInterfaces';
+import { CallGraph, Node } from './callGraph';
+import { SharedEventProperties } from 'types/chromeEventInterfaces';
+
+/* Converts our data into an intermediate form used for processing
+ * For Example: samples => [{
+ *    "cpu": "-1",
+ *    "name": "",
+ *    "ts": "142867379",
+ *    "pid": 6052,
+ *    "tid": "6105",
+ *    "weight": "1",
+ *    "sf": 16
+ *}]. This function converts it to '6052:6105': [{
+ *    "cpu": "-1",
+ *   "name": "",
+ *    "ts": "142867379",
+ *    "pid": 6052,
+ *    "tid": "6105",
+ *    "weight": "1",
+ *    "sf": 16
+ *}]
+ */
+const getPidTid = (data: HermesSample[]) => {
```

Comment:
Returns a dictionary with ``` key = `${processId}:${threadId}` ``` and the value being all the events in the samples with the corresponding process and thread ID